modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/SystemObjectDefinitionMetadataModelListener.java

```diff
@@ -120,15 +124,16 @@ private void _executeObjectActions(
 				_modelClass.getName(), _getCompanyId(baseModel),
 				objectActionTriggerKey,
 				_getPayloadJSONObject(
-					objectActionTriggerKey, originalBaseModel, baseModel),
+					objectActionTriggerKey, originalBaseModel, baseModel,
+					userId),
```

Comment:
Not sure about this userId, but it is not our logic so we can leave it as it is.

What I understand is that we will use the user making the action that triggers the webhook or the userId that creates the baseModel. I think it doesn't make sense but alright.

Anyway I'll try to find a way to pass my concerns to the object team